infrastructure/stacks/service_etl/locals.tf

```diff
@@ -36,6 +36,13 @@ locals {
   service_etl_cloudwatch_event_action          = "lambda:InvokeFunction"
   service_etl_cloudwatch_event_princinple      = "events.amazonaws.com"
 
+  service_etl_sns_name                             = "uec-sf-${var.profile}-service-etl-sns"
```

Comment:
Didn't spot this until now, but this ETL and all the bits surrounding it are part of the fuzzy project (uec-sfs).  Let's not worry about that for this PR and maybe we can create a ticket to refactor the names of these components from 'uec-sf' to 'uec-sfs'. Can you add the ticket as a story to SFD-3380 please?

Reply:
https://nhsd-jira.digital.nhs.uk/browse/SFD-3545